src/sidebar/services/test/auth-test.js

```diff
@@ -137,7 +137,7 @@ describe('AuthService', () => {
     context('when the access token request fails', () => {
       const expectedErr = new Error('Grant token exchange failed');
       beforeEach('make access token requests fail', () => {
-        fakeClient.exchangeGrantToken.returns(Promise.reject(expectedErr));
+        fakeClient.exchangeGrantToken.throws(expectedErr);
```

Comment:
This method is async, so it needs to reject rather than throw.

Reply:
Thank you. I knew something smelled real bad here. I am missing an `await` keyword in my refactored method; I _knew_ tests shouldn't need to change, so I'm glad I asked.